jokes.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "lang": "ar",
+    "joke": "واحد حاب يتزوج على مرتو ومالقاش كيفاه يفهمها\nقالها: ياودي هاد البلاد مافهمنا فيها والو. البارح خرج قانون جديد: لازم كل راجل خدام يتزوج بأكثر من وحدة... ولا يقطعولو راسو!\nقاتلو مربتو: معليش عمري... موت شهيد خير",
+    "img": null,
+    "tags": [ "زواج" ]
+  },
+  {
+    "lang": "ar",
+    "joke": null,
+    "img": "https://scontent-mrs1-1.xx.fbcdn.net/v/t1.0-9/15941503_1856068318002239_6728787835084139089_n.jpg?oh=9fdead02d71b0b87d7c7f2db0c1af2a4&oe=5919D061",
+    "tags": [ "conjugaison", "fr" ]
+  }
+]
```

Comment:
I'll drop those files and use the external dependency instead as @houssem-yahiaoui mentionned